### PR TITLE
Feat/fee 767 delete query

### PR DESCRIPTION
**Pull request by mczapkowicz**
https://metakeen.atlassian.net/jira/software/projects/FEE/boards/10?selectedIssue=FEE-767

### CODE REVIEW

File: src/components/Confirm/Confirm.styles.ts [hunk 1]
```diff
@@ -16,23 +17,43 @@ export const Footer = styled.div`
...
 `;
 
-export const Description = styled.div`
-  margin: 20px 25px;
+export const Description = styled.div<{ isOverflow?: boolean }>`
   width: 350px;
-
   font-size: 16px;
   line-height: 19px;
   font-family: 'Lato Regular', sans-serif;
```
Comment (apryka) [inline src/components/Confirm/Confirm.styles.ts:24]: please remove font-family and use `keen.io/typography` here

File: src/components/Confirm/Confirm.styles.ts [hunk 2]
```diff
@@ -16,23 +17,43 @@ export const Footer = styled.div`
...
 `;
 
 export const Name = styled.strong`
-  display: block;
   max-width: 100%;
   white-space: nowrap;
   overflow: hidden;
   text-overflow: ellipsis;
   line-height: 2;
   font-family: 'Lato Bold', sans-serif;
```
Comment (apryka) [inline src/components/Confirm/Confirm.styles.ts:44]: please remove font-family and use `keen.io/typography` here

File: src/components/Confirm/Confirm.test.tsx [hunk 3]
```diff
@@ -21,6 +23,15 @@ const render = (storeState: any = {}) => {
   };
 };
 
+const intersectionObserverMock = () => ({
```
Comment (apryka) [inline src/components/Confirm/Confirm.test.tsx:26]: you can use built-in util `import { mockAllIsIntersecting } from 'react-intersection-observer/test-utils';`

File: src/components/Confirm/Confirm.tsx [hunk 4]
```diff
@@ -9,42 +10,108 @@ import {
...
+            ) : (
+              <>
+                {t('confirm.delete_message_head')}
+                <Name>{displayName}</Name>
+                {t('confirm.delete_message_tail')}
+                {enableDashboardsConnection && (
+                  <>
+                    {connectedDashboards && connectedDashboards.length > 0 ? (
+                      <div data-testid="connected-dashboards">
+                        <p>{t('confirm.connected_dashboards_info')}</p>
```
Comment (apryka) [inline src/components/Confirm/Confirm.tsx:82]: `<p>` may have some styling assigned in keen-web for example that can break the layout

**Review by apryka — APPROVED** [review]

**Review by dariuszlacheta — APPROVED** [review]

**Pull request merged**